### DISCUSSION TITLE
dodanie-drugiego-polaczenia-internetowego-do-routera-z-openwrt/

### DISCUSSION THREAD

**Discussion by giscus** [idarek/dariusz.wieckiewicz.org#116] [General]
# dodanie-drugiego-polaczenia-internetowego-do-routera-z-openwrt/

Wolny świat, wolne myśly, wolne słowa.

https://dariusz.wieckiewicz.org/dodanie-drugiego-polaczenia-internetowego-do-routera-z-openwrt/

**Comment by lukaszx404** [comment]
dzieki mistrzu wydaje mi sie ze działa ale nie wiem czy tak jak chcialem uzywam portow wwan i usb0 a chcialem zeby pc ciagnal z obydwoch zrodel niewiem czy tak mozna ale jezeli chodzi o poradnik to git wyjasnione

**Reply by idarek** [reply]
Oczywiście że się da. Pakiet MWAN to nie inaczej jak Multi WAN i został stworzony do używania wielu połączeń jednocześnie. Zastosowań jednak jest więcej. W Members ustaw Weight obu interfejsów na 1, i utwórz/o ile nie masz jeszcze, Policies z oboma interfejsami jako podstawową. Następnie Rule. Takim sposobem powinieneś mógł używać obu interfejsów na raz. Kwestia zabawy.
